templates/core/home.html

```diff
@@ -41,7 +41,7 @@ <h1 class="brand"><i class="fa fa-cubes fa-fw"></i> Docs.rs</h1>
                     {%- endif -%}
 
                     <li>
-                        <a href="{{ release_url }}"
+                        <a href="{{ release_url | safe }}"
```

Comment:
This doesn't escape the name or version, is that intentional? I saw that you escaped them later.

Reply:
It should be escaped as a whole, I don't think escaping happens as part of variable building

Reply:
I'm not sure what you mean here. You pipe it to `safe` which means it won't be escaped, which isn't the same behavior as elsewhere.